### ISSUE TITLE
redirect decommissioned sites to archive.org

### ISSUE THREAD

**Issue by afeld** [18F/pages-redirects#148]
Right now, we redirect most of our [decommissioned](https://before-you-ship.18f.gov/infrastructure/decommissioning/) static sites to the corresponding GitHub repositories. I imagine this is confusing to users, especially if they aren't developers, because they wouldn't know where to look for the content, and it's certainly not easy to use since the navigation and layout is gone. A better experience would be to redirect to the last (good) capture on archive.org, at either

- `https://web.archive.org/[URL]` - latest capture
- `https://web.archive.org/web/[timestamp]/[URL]` - specific point in time

archive.org isn't a government site, but I don't think that means we can't do it. Let me know if you know otherwise.

cc @amirbey, [related "Sunset Vault" 10x project](https://trello.com/c/gZTZcR22/52-the-sunset-vault) where this idea came up

**Comment by dawnpm** [comment]
Before redirecting in this manner, site manager has to confirm that archive.org actually has their site captured. They don't have everything.

**Comment by afeld** [comment]
Good call! Checking for that and [triggering an archive](https://help.archive.org/hc/en-us/articles/360001513491-Save-Pages-in-the-Wayback-Machine) if necessary should be part of the steps.

**Comment by its-a-lisa-at-work** [comment]
@afeld is this something that we're doing or that we're watching? If we're doing, could we move over to the TTS Tech Portfolio repo for better issue tracking?
